pkg/snet/snet.go

```diff
@@ -154,6 +154,16 @@ func (n *SCIONNetwork) Dial(ctx context.Context, network string, listen *net.UDP
 		return nil, err
 	}
 	log.FromCtx(ctx).Debug("UDP socket opened on", "addr", packetConn.LocalAddr(), "to", remote)
+
+	// TODO: make STUN handling optional/configurable
```

Comment:
Does this TODO need to be addressed before merging the PR? If not, we should remember to open an issue for this.